packages/fast-colors/src/color-converters.ts

```diff
@@ -16,18 +16,6 @@ import { degreesToRadians, radiansToDegrees } from "./math-utilities";
 // https://web.stanford.edu/~sujason/ColorBalancing/adaptation.html
 // http://brucelindbloom.com/index.html
 
-/**
- * Get the relative luminance of a color.
- * Adjusts the color to sRGB space, which is necessary for the WCAG contrast spec.
- * The alpha channel of the input is ignored.
- * @param rgb The input color
- *
- * @deprecated Use rgbToRelativeLuminance instead.
- */
-export function rgbToLuminance(rgb: ColorRGBA64): number {
-    return rgbToRelativeLuminance(rgb);
-}

```

Comment:
should we wait for a breaking change before removing this?

Reply:
this *is* a breaking change